migrations/v3.js

```diff
@@ -0,0 +1,133 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+const getCourse = content => {
+  const [course] = content.filter(({ _type }) => _type === 'course');
+  return course;
+};
+
+const getGlobals = content => {
+  return getCourse(content)?._globals?._extensions?._glossary;
+};
+
+describe('Glossary - v2.1.3 to v3.0.0', async () => {
+  // https://github.com/adaptlearning/adapt-contrib-glossary/compare/v2.1.3..v3.0.0
+
+  let course, courseGlossaryGlobals;
+
+  whereFromPlugin('Glossary - from v2.1.3', { name: 'adapt-contrib-glossary', version: '<3.0.0' });
+
+  whereContent('Glossary - where configured', async (content) => {
+    course = getCourse(content);
+    if (course._glossary) return true;
+  });
+
+  mutateContent('Glossary - add globals if missing', async (content) => {
+    courseGlossaryGlobals = getGlobals(content);
+    if (courseGlossaryGlobals) return true;
+    course._globals._extensions = course._globals._extensions || {};
+    courseGlossaryGlobals = course._globals._extensions._glossary = {};
+    return true;
+  });
+
+  mutateContent('Glossary - add new globals', async (content) => {
```

Comment:
Whilst this does work, from a best practise perspective it would be better to split these up, so having a mutate and check for each attribute seperately. It does add to the code but it does make the logs much easier to read in case of an issue.